Anki 24.06 changes.md (#653)
open-spaced-repetition/fsrs4anki: docs/tutorial.md

The FSRS optimizer uses machine learning to learn your memory patterns and find parameters that best fit your review history. So, the optimizer requires several reviews to fine-tune the parameters.

> If you have less than 1000 reviews (400 in Anki 24.04) across all cards that this preset is applied to, please use the default parameters that are already entered into the "FSRS parameters" field. Even with the default parameters, FSRS is better than the default Anki algorithm (SM-2). In Anki 24.06, the limit has been removed. You can now use "Optimize" regardless of how many reviews you have, though if the number of reviews is small, you will likely see "The FSRS parameters currently appear to be optimal". Instead of a hard limit, there is now a complex rule that determines what will happen when you click "Optimize": default parameters will be kept; only the first 4 parameters will be optimized and the rest will be set to default; or all parameters will be optimized. 

Author: I think it's fine. It's better to document this to avoid people saying "This behavior is not mentioned anywhere"

***

Q5: I'm sure I have >1000 reviews (>400 in Anki 24.04), yet when I try to optimize parameters for my preset, I get an error telling me that I don't have enough reviews. Is that a bug?

> A5: FSRS only takes into account one review per day. If you review a card multiple times per day, only the chronologically first review will be used by the optimizer. Also, if your deck has subdecks, ensure that the preset is applied to the subdecks, not just to the parent deck. Also, if you are using Anki 24.06 or newer, you can use "Optimize" with any number of reviews.

Reviewer: ```suggestion
A5: FSRS only takes into account one review per day. If you review a card multiple times per day, only the chronologically first review will be used by the optimizer. Also, if your deck has subdecks, ensure that the preset is applied to the subdecks, not just to the parent deck. Note that in Anki 24.06 and newer, there is no minimum limit for the number of reviews to use "Optimize".
```

A18: The default parameters are generated from 20k collections. They are the median values of 20k sets of parameters. Thus, inevitably, half of the new users will find that their retention is lower than desired retention, while the other half will discover that their retention exceeds their desired retention.

> If you have enough reviews (at least 1000/400 reviews), you can optimize FSRS parameters to personalize them for yourself. If you don't have enough reviews and the true retention is significantly lower than your desired retention, I recommend increasing the desired retention until you have enough reviews for optimization. You can check your true retention using the Helper add-on, just Shift + Left Mouse Click on Stats.

Author: But this is a question specifically about default parameters

> If the true retention is significantly lower than your desired retention, it is recommended to increase desired retention. You can check your true retention using the FSRS4Anki Helper add-on, just Shift + Left Mouse Click on Stats. Of course, if you have a large number of reviews, you should use optimized parameters instead of the default parameters.

Reviewer: ```suggestion
If the true retention is significantly lower than your desired retention, try increasing the desired retention to compensate. You can check your true retention using the FSRS4Anki Helper add-on, just Shift + Left Mouse Click on Stats. Of course, if you have a large number of reviews, it is advisable to use the optimized parameters rather than the default parameters.
```
